api/models/source_logger.py

```diff
@@ -34,3 +40,16 @@ class WebHookLogger(OptionalSectionConfig):
 
 class SourceLoggerListDTO(SnakeModel):
     sourcesLoggers: List[SourceLoggerDTO]
+
+
+def validate_logger(logger: SourceLoggerDTO):
```

Comment:
weren't these also supposed to return the class in order to create the object?

Reply:
Or creating the object is just for validation purposes and  the config is edited regardless of the created object?

Reply:
Creating the object is just for validation purposes (for example, a mandatory field is not sent.)